### DISCUSSION TITLE
Gallery

### DISCUSSION THREAD

**Discussion by erikdubois** [leftwm/leftwm#657] [Ideas]
I get more and more question of people asking what system I am working on.
Maybe it is a good idea to start a gallery - a more visual way of showing what you super coders are up to.

The one image does not give enough credit to the awesome twm you are building.

Keep up the good work.

Just a screenshot of todays look.

![ArcoLinux-2022-01-11_09-16](https://user-images.githubusercontent.com/10594806/148905573-e2e79ba7-2418-40f3-8011-82bb91befeba.png)

**Comment by erikdubois** [comment]
all the themes we have in one video

https://youtu.be/DTwdoM4dMfw

**Comment by VuiMuich** [comment]
This is so great and thanks a lot for your praise!
Indeed some kind of gallery might be nice, maybe as part of the leftwm/leftwm-contrib repo?
